### ISSUE TITLE
Flow's type declarations for class fields are not completely stripped

### ISSUE THREAD

**Issue by ide** [babel/babel#1686]
Field declarations for flow inside of classes are not completely stripped, producing invalid syntax.

input file
```js
/**
 * @flow
 */
type Options = {};

export default class Plugin {
  options: Options;
  constructor(options: Options) {
    this.options = options;
  }
}
```

.babelrc
```json
{
  "whitelist": [
    "flow",
    "strict",
    "es6.arrowFunctions",
    "es6.destructuring",
    "es6.modules",
    "es7.classProperties"
  ]
}
```

output file
```js
/**
 * @flow
 */
'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});

class Plugin {
  options;

  constructor(options) {
    this.options = options;
  }
}

exports['default'] = Plugin;
module.exports = exports['default'];
```

I believe this bit here is invalid and doesn't run on io.js.
```js
class Plugin {
  options;
```

**Comment by ide** [comment]
Thanks for the super quick fix @sebmck! For context I omitted the class transformer because I was targeting io.js which supports the class keyword natively now, so it's quite nice to prune the transformer whitelist as V8/io.js make progress =)

**Comment by sebmck** [comment]
Ah yeah, understood. Fixed as of 5.5.1, thanks!
